manager/src/main/java/com/redhat/service/smartevents/manager/dao/BridgeDAO.java

```diff
@@ -38,6 +38,12 @@ public Bridge findByIdAndCustomerId(String id, String customerId) {
         return find("#BRIDGE.findByIdAndCustomerId", params).firstResult();
     }
 
+    public List<Bridge> findByOrganisationId(String organisationId) {
+        Parameters params = Parameters
+                .with("organisationId", organisationId);
+        return find("#BRIDGE.findByOrganisationId", params).list();
```

Comment:
@r00ta Can we use `count()` instead of `list()` and return a `long` from this method? We load all the bridges from the DB only to call `size()` on the returned list.

Reply:
done